core/scss/utilities/mixins/typography/_displays.scss

```diff
@@ -0,0 +1,20 @@
+@charset "UTF-8";
+
+//
+// @displays
+//
+// Default displays shared styling.
+//
+// Style guide: Mixins.Typography.displays
+//
+@mixin displays {
+  @include margin(0 null 0.5em);
```

Comment:
```suggestion
  @include margin(0 null null);
```

Reply:
Are we adding margin-bottom to all headers? I like the margin-top:0 since that's like a reset, but perhaps we could leave out the margin-bottom? In cards, for example, the h3 and h5 have custom margin-bottoms using modular-spacing. Or perhaps you meant this to be part of the vertical rhythm?

Reply:
Originally, I think this was an attempt at vertical rhythm as it relates to headers. That said, the use case in which you've brought up seems to be the route we've decided to go (at least until we have the time and effort to put into more global vertical rhythm). That said, I'll drop the margin bottom.